src/scss/grommet-core/_objects.card.scss

```diff
@@ -39,7 +39,14 @@ $card-hover-color: #EBEBEB;
     }
   }
 
-  // &__thumbnail {
+  &.#{$grommet-namespace}box--responsive {
```

Comment:
I would have thought we got this behavior automatically from box. How is this changing it?